codalab/worker/docker_utils.py

```diff
@@ -142,6 +142,11 @@ def start_bundle_container(
 
     # Name the container with the UUID for readability
     container_name = 'codalab_run_%s' % uuid
+
+    if container_exists(container_name):
+        client.containers.remove(container_name)
```

Comment:
What happens if this fails?

Reply:
This error was thrown from [_transition_from_PREPARING](https://github.com/codalab/codalab-worksheets/blob/master/codalab/worker/local_run/local_run_state.py#L153). I searched through the entire worker code base, didn't find any trace indicating that it's trying to restart the same container twice. Maybe @bkgoksel would have some suggestions?